### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by jshaughn**

### CODE REVIEW

File: hawkular-alerts-ui/src/main/ui/bower.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "hawkular-alerts-ui",
+  "version": "2.0.0",
```
Comment (lucasponce) [inline hawkular-alerts-ui/src/main/ui/bower.json]: Minor, this is just informational and it doesn't mean to deploy anything, but I guess we should put here a 1.8.0.

File: hawkular-alerts-ui/src/main/ui/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "hawkular-alerts-ui",
+  "version": "2.0.0",
```
Comment (lucasponce) [inline hawkular-alerts-ui/src/main/ui/package.json]: Same here

**Comment by lucasponce** [comment]
A couple of minor comments.
Also I have a suggestion to put the version of the hawkular alerting somewhere in the UI dashboard, as we have 1.8.x and 2.0, it could be good to have some minor text on the UI.

File: hawkular-alerts-ui/src/main/ui/bower.json (hunk 1 above)
Comment (jshaughn) [reply hawkular-alerts-ui/src/main/ui/bower.json]: To be honest, this manually maintained versioning is unlikely to stay updated.  But at least we can start with a reasonable version.  I'll change.

**Comment by jshaughn** [comment]
@lucasponce I updated the version in the two files.  More importantly, I added an "About..." option to the dash which pops up an about box showing our /status info.

**Comment by lucasponce** [comment]
thanks

**Pull request merged**